locales/english.json

```diff
@@ -19,6 +19,13 @@
   "frontpage.disclaimer": "Data Marketplace is currently in beta. Some functionality might be limited, but we\u0027re constantly working to improve your experience.",
   "frontpage.popularProductsHeader": "Popular data",
   "frontpage.numberOfViews": "{numberOfViews} views",
+  "frontpage.relevantdata.header": "Relevant Information for you",
+  "frontpage.relevantdata.data.tomany": "Data to the Many",
+  "frontpage.relevantdata.data.tomany.text": "Data to the Many is an initiative aiming to make Equinor a data-driven organization. Read about the initiative and how it can help improve your work here.",
+  "frontpage.relevantdata.data.analytics": "Data Analytics Community",
+  "frontpage.relevantdata.data.analytics.text": "An arena for information sharing, knowledge exchange and collaboration between anyone who are working with or are interested in data analytics across tools and domains.",
+  "frontpage.relevantdata.data.product": "What is Data Product?",
+  "frontpage.relevantdata.data.product.text": "Curious about Data Products, and how it can improve your work day? Enterprise Data Management’s educational videos gives an overview of Data Products and related topics. ",
```

Comment:
i would recommend revisiting these property names to better reflect the sections they belong to